rpcs/ledgerService.go

```diff
@@ -259,6 +308,28 @@ func (ls *LedgerService) sendCatchupRes(ctx context.Context, target network.Unic
 	}
 }
 
+func topicBlockBytes(ledger *data.Ledger, round basics.Round, requestType string) network.Topics {
+	blk, cert, err := ledger.EncodedBlockCert(round)
+	if err != nil {
+		errMsg := "LedgerService topicBlockBytes: error in EncodedBlockCert: " + err.Error()
+		return network.Topics{
+			network.MakeTopic(network.ErrorKey, []byte(errMsg))}
+	}
+	switch requestType {
+	case blockAndCertValue:
+		return network.Topics{
+			network.MakeTopic(
+				blockDataKey, blk),
+			network.MakeTopic(
+				certDataKey, cert),
+		}
+	default:
+		errMsg := "LedgerService topicBlockBytes: request type is unknown"
```

Comment:
nit: use the same "language" that the caller has used. i.e. `requestedDataType specifies unsupported data type`